ui-admin/src/study/surveys/ConsentView.tsx

```diff
@@ -55,32 +57,65 @@ function RawConsentView({ studyEnvContext, consent, readOnly = false }:
       readOnly={readOnly}
       onCancel={() => navigate(studyEnvFormsPath(portal.shortcode, study.shortcode, currentEnv.environmentName))}
       onSave={createNewVersion}
+      isConsentForm={true}
     />
   )
 }
 
 export type ConsentParamsT = StudyParams & {
-  consentStableId: string
+  consentStableId: string,
+  version: string
+}
+
+/** loads a consentForm */
+export const useLoadedConsentForm = (portalShortcode: string, stableId: string, version: number) => {
+  const [form, setForm] = useState<ConsentForm | undefined>()
```

Comment:
Just for my information... What does `useState <ConsentForm | undefined>()` mean, and how is it different from `useState <ConsentForm>({})`?

Reply:
it's the same -- will fix, thanks